obs-packaging/download_image.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+handle_error() {
+	local exit_code="${?}"
+	local line_number="${1:-}"
+	echo "Failed at $line_number: ${BASH_COMMAND}"
+	exit "${exit_code}"
+}
+trap 'handle_error $LINENO' ERR
+
+BRANCH=${1:-master}
+
+echo "Download kata image from branch ${BRANCH}"
+
+if ! version=$(curl -s --fail -L https://raw.githubusercontent.com/kata-containers/runtime/${BRANCH}/VERSION);then
+	echo "failed to get version from branch ${BRANCH}"
```

Comment:
Nit: error messages should be directed to `stderr`, so maybe add a `die()` function? That would make the code easier to read as you wouldn't need all the `exit 1` calls too.